### PR TITLE
{lib}[gompi/2020a,iimpi/2020a] OpenImageIO v2.1.12.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node371.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/7eeaa643a3e1ce1fee485e4d5ad16c15 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2602.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/44e3ea5b1973264175fd994fa082bdd5 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node123.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/2556ec9ef7ca7b14373241f3f18e36ce for a full test report.

File: easybuild/easyconfigs/o/OpenImageIO/OpenImageIO-2.1.12.0-gompi-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+dependencies = [
+    ('Boost', '1.72.0'),
+    ('libjpeg-turbo', '2.0.4'),
+    ('libpng', '1.6.37'),
+    ('LibTIFF', '4.1.0'),
+    ('OpenEXR', '2.4.1'),
+    ('freetype', '2.10.1'),
+    ('zlib', '1.2.11'),
+]
+builddependencies = [('CMake', '3.16.4')]
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenImageIO/OpenImageIO-2.1.12.0-gompi-2020a.eb:25]: I should also add `git` as a build dep here, see #11328

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/cf759162e3611f34b2101731e1df810b for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node128.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/0b6cbf8f069e6751431b5017e92eae7c for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**